docs/lib/graphqlapi/fragments/flutter/advanced-workflows/30_nested.md

```diff
@@ -0,0 +1,31 @@
+```dart
+try{
+    String graphQLDocument = '''query getPost(\$id: ID!) {
+        "getTodo(id: \$id) {
+            "id
+            "title
+            "rating
+            "status
+            "comments {
+                "items {
+                    "id
+                    "postID
+                    "content
+                }
+            }
+        }
+    }''';
+
+    var operation = await Amplify.API
+        .query<String>(request: GraphQLRequest(
+        document: graphQLDocument,
+        variables: {"id": "[TODO_ID]"});
```

Comment:
Are all of these examples tested? There is one closing parenthesis missing and the indentation looks off.

```suggestion
    var operation = Amplify.API.query<String>(
        request: GraphQLRequest(
            document: graphQLDocument, variables: {"id": "[TODO_ID]"}));
```

Reply:
Thanks updated to add missing )  

Ashish will add updated examples